packages/react-ui-components/src/SelectBox/selectBox.js

```diff
@@ -2,8 +2,10 @@ import React, {PureComponent} from 'react';
 import PropTypes from 'prop-types';
 import DropDown from '../DropDown/index';
 import DefaultSelectBoxOption from './defaultSelectBoxOption';
+import keydown from 'react-keydown';
 import mergeClassNames from 'classnames';
 
+@keydown
```

Comment:
Please have a look at https://github.com/glortho/react-keydown#monitor-only-key-codes-which-you-care-about

I think it would be better to restrict this. During inline editing we have tons of key events, that would trigger a redraw of every select box otherwise.

Reply:
Hey @grebaldi I've updated my PR.
The events were only be captured when the selectbox is active but you are right, to restrict them would be better anyway.